extensions/wikia/ImageReview/ImageReviewEventsHooks.class.php

```diff
@@ -1,11 +1,19 @@
 <?php
 
-use \Wikia\Logger\WikiaLogger;
-use \Wikia\Tasks\Tasks\ImageReviewTask;
+use Wikia\Logger\WikiaLogger;
+use Wikia\Rabbit\ConnectionBase;
+use Wikia\Tasks\Tasks\ImageReviewTask;
 
 class ImageReviewEventsHooks {
 	public static function onUploadComplete( UploadBase $form ) {
-		static::createAddTask( $form->getTitle() );
+		global $wgCityId, $wgImageReviewTestCommunities;
+
+		// TODO: community switch
+		//if ( in_array( $wgCityId, $wgImageReviewTestCommunities ) ) {
```

Comment:
Let's have a default for `$wgImageReviewTestCommunities` — `[ 177 ]` (Community Central).  And optionally, `mediawiki119` (can't remember the ID). I mean let's complete this implementation.